handbook/product/technical_writing/windows_development.md

```diff
@@ -0,0 +1,78 @@
+# Using a windows environment
```

Comment:
@davejrt and @pecigonzalo - you were both mentioned as using PC's in the guide. Is the information on the Windows doc helpful to you at all? Do you think it will be useful to other teammates? Going forward, anyone on the product team is expected to be running a unix environment (windows is not acceptable).

Reply:
WSL2 Is a UNIX environment, I would not be working directly over Windows, but I think its fully acceptable to use a Windows PC with WSL.

Reply:
Ok, makes sense. I think your recommendation to always use a Linux environment and just follow those instructions is sufficient. I don't want to keep documentation that will lead to more confusion. I'll delete this page.

Thank you for all the clarification!